src/language/language_type.tera.rs

```diff
@@ -54,8 +54,13 @@ impl LanguageType {
     }
 
     /// Provides every variant in a Vec
-    pub fn list() -> &'static [Self] {
-        &[{% for key, _ in languages %}{{key}}, {%- endfor %}]
+    pub fn list() -> &'static [(Self, &'static [&'static str])] {
```

Comment:
Since this changes the return type of `LanguageType::list`, it is a breaking change for users that currently use tokei as a library. It might be better to create a new `fn` instead.

Reply:
The breaking change is fine, I intend to release a new major version.

Reply:
Sorry for my late reply, and thank both of you for your review!